shinatra.sh

```diff
@@ -1,5 +1,2 @@
 #!/bin/bash
-RESPONSE="HTTP/1.1 200 OK\r\nConnection: keep-alive\r\n\r\n${2:-"OK"}\r\n"
-while { echo -en "$RESPONSE"; } | nc -l "${1:-8080}"; do
-  echo "================================================"
-done
+python -m SimpleHTTPServer "${1:-8080}"
```

Comment:
That's true only if you have Python.. and, more specifically, Python 2. Python 3 replaced SimpleHTTPServer with http.server.

It's true that, alternatively, you have to have netcat if you want to use this.. but most distros do (except for Arch by default.. jeez, wtf?)